model/converter/thrift/jaeger/from_domain.go

```diff
@@ -103,6 +103,7 @@ func (d domainToJaegerTransformer) keyValueToTag(kv *model.KeyValue) *jaeger.Tag
 func (d domainToJaegerTransformer) convertKeyValuesToTags(kvs model.KeyValues) []*jaeger.Tag {
 	jaegerTags := make([]*jaeger.Tag, len(kvs))
 	for idx, kv := range kvs {
+		kv := kv
 		jaegerTags[idx] = d.keyValueToTag(&kv)
```

Comment:
```suggestion
	for i := range kvs {
		jaegerTags[i] = d.keyValueToTag(&kvs[i])
```

Reply:
Same comment as above, just want to confirm that the intention is to use the reference to the object itself and not a copy. 
Although `kvs` is passed in as an arg so it is already being passed by value